### PR TITLE
NMS-12917: Update Minion installation procedures

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [x ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ x] Have you made a comment in that issue which points back to this PR?
* [ x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12917

### CODE REVIEW

**Comment by Bonrob2** [comment]
New and improved, I hope. 
Still no documentation for scenario of ActiveMQ broker running on a dedicated server independently from Horizon Core, but if it's important we can create a new JIRA issue to capture it. 
Also, will eventually go through the entire installation guide to clean it up.

File: opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/rhel/step-1-install-minion.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+==== Step1: Install the repository and Minion package
```
Comment (indigo423) [inline opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/rhel/step-1-install-minion.adoc:2]:
```suggestion
==== Step 1: Install the repository and Minion package
```

File: opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/setting-up-minion.adoc [hunk 2]
```diff
@@ -1,34 +1,36 @@
 
-== Monitor isolated location with Minion
+== Installing and Configuring a Minion
 
-This section describes how to install the _Minion_ to monitor devices and services in a location which can't be reached from an _{opennms-product-name}_ instance.
+A Minion is an instance of the Karaf OSGi service that enables OpenNMS to monitor devices and services in locations that an OpenNMS instance cannot reach. Minions communicate with these remote devices while OpenNMS performs coordination and task delegation.
 
-=== Objectives
+Minions can operate behind a firewall and/or network address translation (NAT) as long as they can communicate with OpenNMS via ActiveMQ, Apache Kafka, or gRPC.
```
Comment (indigo423) [inline opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/setting-up-minion.adoc:6]: No worries here, just the way where we don't have a name for the Server component. What "Meridian Core" is in our commercial pricing is "OpenNMS Instance" or just "OpenNMS" here or "OpenNMS Horizon" or just "Horizon" in other places. In the Cloud Portal we just call it "Instance".

So the "server" / "main application component" has the following names and means the same thing:

* Meridian
* Meridian Core
* OpenNMS Horizon
* Horizon
* OpenNMS Instance
* Instance
* OpenNMS

**Review by indigo423 — APPROVED** [review]
LGTM! 👍 Just found a space typo and everything else I would file under food for thoughts :)

File: opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/setting-up-minion.adoc (hunk 2 above)
Comment (Bonrob2) [reply opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/setting-up-minion.adoc:6]: Agreed. We need to standardize this. Let's decide on what to call it, and then revise all the documentation to be consistent. 
We could start a glossary of terms on Confluence to capture this.

**Pull request merged**